integration_tests/steps/kpi.py

```diff
@@ -85,6 +85,10 @@ def penalty_rate_respected(context, device_name):
     assert isclose(bills[str(device.uuid)]["penalty_cost"],
                    bills[str(device.uuid)]["penalty_energy"] * DEVICE_PENALTY_RATE / 100.0,
                    rel_tol=0.0003 * DEVICE_PENALTY_RATE)
+    assert isclose(bills[str(house1.uuid)]["Accumulated Trades"]["penalty_cost"],
```

Comment:
Just that it caught my attention now, having a two word dictionary key (Accumulated Trades) doesn't seems like a good approach. No need to do anything for now as its been there for a while

Reply:
@faizan2590 this key is used by the UI as a row header, that's why it is a string here. It would be great if it remained as is, in order to avoid crashing the UI